spec/models/user_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+  # let (:user) { User.new(first_name:'j',email:'p@m.com',password:'foobar',password_confirmation:'foobar').save}
+  # it 'validates first name' do 
+  #   expect(user).not_to eq(true)
+  # end
+
+  it 'validates last name' do 
```

Comment:
It appears we have some redundant validations here. I think we can safely delete this one